lib/samples/edit_feature_attachments/edit_feature_attachments_sample.dart

```diff
@@ -0,0 +1,379 @@
+      _featureLayer,
+      screenPoint: localPosition,
+      tolerance: 5.0,
+      maximumResults: 1,
+    );
+
+    // If there are features identified, show the bottom sheet to display the
+    // attachment information for the selected feature.
+    final features =
+        identifyLayerResult.geoElements.whereType<Feature>().toList();
+    if (features.isNotEmpty) {
+      _featureLayer.selectFeatures(features: features);
+      final selectedFeature = features.first as ArcGISFeature;
+      if (mounted) _showBottomSheet(selectedFeature);
+    }
+  }
+
+  // Show the bottom sheet to display the attachment information.
+  void _showBottomSheet(ArcGISFeature selectedFeature) {
+    showModalBottomSheet(
+      context: context,
+      builder: (context) => AttachmentsOptions(
+        arcGISFeature: selectedFeature,
+        applyEdits: _applyEdits,
+      ),
+    );
+  }
+
+  // Apply the changes to the feature table.
+  Future<void> _applyEdits(ArcGISFeature selectedFeature) async {
+    final serviceFeatureTable =
+        _featureLayer.featureTable! as ServiceFeatureTable;
+    try {
+      // Update the selected feature locally.
+      await serviceFeatureTable.updateFeature(selectedFeature);
+      // Apply the edits to the service.
+      await serviceFeatureTable.applyEdits();
+    } catch (e) {
+      if (mounted) _showErrorDialog(context, e);
+    }
+    return Future.value();
+  }
+}
+
+//
+// A widget to display the attachment information for the selected feature.
+//
+class AttachmentsOptions extends StatefulWidget {
+  final ArcGISFeature arcGISFeature;
+  final Function(ArcGISFeature) applyEdits;
+
+  const AttachmentsOptions({
+    super.key,
+    required this.arcGISFeature,
+    required this.applyEdits,
+  });
+
+  @override
+  State<AttachmentsOptions> createState() => _AttachmentsOptionsState();
+}
+
+// State class for the AttachmentsOptions.
+class _AttachmentsOptionsState extends State<AttachmentsOptions>
+    with SampleStateSupport {
+  late final String damageType;
+  var attachments = <Attachment>[];
+  var isLoading = false;
+
+  @override
+  void initState() {
+    super.initState();
+    damageType = widget.arcGISFeature.attributes['typdamage'];
+    _loadAttachments();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: [
+        Column(
+          children: [
+            // Display the damage type and a close button.
+            Container(
+              color: Colors.purple,
+              padding: const EdgeInsets.fromLTRB(5, 0, 5, 0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Text(
+                    'Damage Type: $damageType',
+                    style: const TextStyle(
+                        fontSize: 18,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.white),
```

Comment:
This should have a trailing comma after `color: Colors.white` and before the `)`